app/src/main/java/com/lagradost/cloudstream3/extractors/PlayLtXyz.kt

```diff
@@ -0,0 +1,84 @@
+package com.lagradost.cloudstream3.extractors
+
+import android.util.Log
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.lagradost.cloudstream3.app
+import com.lagradost.cloudstream3.utils.*
+import com.lagradost.cloudstream3.utils.AppUtils.tryParseJson
+
+class PlayLtXyz: ExtractorApi() {
+    override val name: String = "PlayLt"
+    override val mainUrl: String = "https://play.playlt.xyz"
+    override val requiresReferer = true
+
+    private data class ResponseData(
+        @JsonProperty("data") val data: String? = null
+    )
+
+    override suspend fun getUrl(url: String, referer: String?): List<ExtractorLink> {
+        val extractedLinksList = mutableListOf<ExtractorLink>()
+        //Log.i(this.name, "Result => (url) $url")
+        var idUser = ""
+        var idFile = ""
+        var bodyText = ""
+        val doc = app.get(url, referer = referer).document
+        //Log.i(this.name, "Result => (url, script) $url / ${doc.select("script")}")
+        doc.select("script").forEach {
+            val text = it?.toString() ?: ""
+            if (text.isNotBlank()) {
+                if (text.contains("var idUser")) {
+                    bodyText = text
+                    //Log.i(this.name, "Result => (bodyText) $bodyText")
+                }
+            }
+        }
+        //Log.i(this.name, "Result => (bodyText) $bodyText")
+        if (bodyText.isNotBlank()) {
+            idUser = "(?<=var idUser = \")(.*)(?=\";)".toRegex().find(bodyText)
+                ?.groupValues?.get(0) ?: ""
+
+            idFile = "(?<=var idfile = \")(.*)(?=\";)".toRegex().find(bodyText)
+                ?.groupValues?.get(0) ?: ""
+        }
+        //Log.i(this.name, "Result => (idUser, idFile) $idUser / $idFile")
+        if (idUser.isNotBlank() && idFile.isNotBlank()) {
+            //val sess = HttpSession()
+            val ajaxHead = mapOf(
+                Pair("Origin", mainUrl),
```

Comment:
you can use 
"Origin" to mainUrl instead of Pair() just FYI :+1:

Reply:
I know but Pair() is more readable to me. (Curse of C# and Python)